### PR TITLE
Epheph/ch11377/claimable eth balance does not show share

### PR DESCRIPTION
**Pull request by epheph**

### CODE REVIEW

File: src/server/getters/get-reporting-fees.ts [hunk 1]
```diff
@@ -362,42 +364,42 @@ function getParticipationTokenEthFees(db: Knex, augur: Augur, reporter: Address,
...
   });
-  participantQuery.leftJoin("token_supply as feeTokenSupply", "feeTokenSupply.token", "fee_windows.feeToken");
-  participantQuery.leftJoin("token_supply as participationTokenSupply", "participationTokenSupply.token", "fee_windows.feeWindow");
+  participantQuery.leftJoin("token_supply as participationTokenSupply", "participationTokenSupply.token", "feeToken.feeWindow");
   participantQuery.where("all_participants.universe", universe);
   participantQuery.where("all_participants.reporter", reporter);
   participantQuery.whereRaw("(reportingState IN (?, ?) OR disavowed IN (?, ?))", [ReportingState.AWAITING_FINALIZATION, ReportingState.FINALIZED, 1, 2]);
   participantQuery.asCallback((err: Error|null, participantEthFeeRows: Array<ParticipantEthFeeRow>) => {
     if (err) return callback(err);
+    console.log("HH", participantEthFeeRows);
```
Comment (nuevoalex) [inline src/server/getters/get-reporting-fees.ts]: I have the same debugging strategy!

File: src/server/getters/get-reporting-fees.ts [hunk 2]
```diff
@@ -362,42 +364,42 @@ function getParticipationTokenEthFees(db: Knex, augur: Augur, reporter: Address,
...
+      .andOn("feeToken.symbol", db.raw("?", "FeeToken"));
   });
-  participantQuery.leftJoin("fee_windows", "fee_windows.feeToken", "feeToken.token");
   participantQuery.leftJoin("balances AS cashFeeWindow", function () {
     this
-      .on("cashFeeWindow.owner", db.raw("fee_windows.feeWindow"))
+      .on("cashFeeWindow.owner", db.raw("feeToken.feeWindow"))
       .on("cashFeeWindow.token", db.raw("?", augur.contracts.addresses[augur.rpc.getNetworkID()].Cash));
   });
+  console.log(participantQuery.toSQL())
```
Comment (nuevoalex) [inline src/server/getters/get-reporting-fees.ts]: remove consoles

Comment (epheph) [reply src/server/getters/get-reporting-fees.ts]: thx!

File: src/server/getters/get-reporting-fees.ts (hunk 1 above)
Comment (epheph) [reply src/server/getters/get-reporting-fees.ts]: Oh man, i thought i got rid of these 😬

**Pull request merged**